examples/counter-next/shared/src/lib.rs

```diff
@@ -0,0 +1,57 @@
+mod app;
+mod capabilities;
+
+use crux_core::bridge::Bridge;
+pub use crux_core::Core;
+pub use crux_http as http;
+use std::sync::LazyLock;
+
+pub use app::*;
+pub use capabilities::sse;
+
+static CORE: LazyLock<Bridge<App>> = LazyLock::new(|| Bridge::new(Core::new()));
+
+#[cfg(not(target_family = "wasm"))]
+const _: () = assert!(
+    uniffi::check_compatible_version("0.29.2"),
+    "please use uniffi v0.29.2"
+);
+
+#[cfg(not(target_family = "wasm"))]
+uniffi::setup_scaffolding!();
+
+/// Ask the core to process an event
+/// # Panics
+/// If the core fails to process the event
+#[cfg_attr(not(target_family = "wasm"), uniffi::export)]
+#[cfg_attr(target_family = "wasm", wasm_bindgen::prelude::wasm_bindgen)]
+pub fn process_event(data: &[u8]) -> Vec<u8> {
+    match CORE.process_event(data) {
+        Ok(effects) => effects,
+        Err(e) => panic!("{e}"),
+    }
+}
+
+/// Ask the core to handle a response
+/// # Panics
+/// If the core fails to handle the response
+#[cfg_attr(not(target_family = "wasm"), uniffi::export)]
+#[cfg_attr(target_family = "wasm", wasm_bindgen::prelude::wasm_bindgen)]
+pub fn handle_response(id: u32, data: &[u8]) -> Vec<u8> {
+    match CORE.handle_response(id, data) {
+        Ok(effects) => effects,
+        Err(e) => panic!("{e}"),
+    }
+}
+
+/// Ask the core to render the view
+/// # Panics
+/// If the view cannot be serialized
+#[cfg_attr(not(target_family = "wasm"), uniffi::export)]
+#[cfg_attr(target_family = "wasm", wasm_bindgen::prelude::wasm_bindgen)]
+pub fn view() -> Vec<u8> {
+    match CORE.view() {
+        Ok(view) => view,
+        Err(e) => panic!("{e}"),
+    }
+}
```

Comment:
Ah, answers my other questions - this is where it still depends on uniffi directly.